### ISSUE TITLE
add quantized model

### ISSUE THREAD

**Issue by 0xdeafc0de** [0xdeafc0de/perceptron#2]
Provide a quantized model with a default scale to do following tasks after training -

- Save original and quantized model
- Evaluate both on the same test input(s)
- Print softmax probabilities and optionally accuracy on a test set
